visualization/app/codeCharta/state/store/appSettings/isAttributeSideBarVisible/isAttributeSideBarVisible.actions.ts

```diff
@@ -0,0 +1,45 @@
+import { Action } from "redux"
+
+export enum IsAttributeSideBarVisibleActions {
+	SET_IS_ATTRIBUTE_SIDE_BAR_VISIBLE = "SET_IS_ATTRIBUTE_SIDE_BAR_VISIBLE",
+	OPEN_ATTRIBUTE_SIDE_BAR = "OPEN_ATTRIBUTE_SIDE_BAR",
+	CLOSE_ATTRIBUTE_SIDE_BAR = "CLOSE_ATTRIBUTE_SIDE_BAR"
+}
+
+export interface SetIsAttributeSideBarVisibleAction extends Action {
+	type: IsAttributeSideBarVisibleActions.SET_IS_ATTRIBUTE_SIDE_BAR_VISIBLE
+	payload: boolean
+}
```

Comment:
I'd remove this if we have opening and closing anyway. 
also good job on choosing some less technical terms for the action here